modules/tinymce/src/core/main/ts/api/html/Schema.ts

```diff
@@ -478,7 +478,8 @@ function Schema(settings?: SchemaSettings): Schema {
     'noshade nowrap readonly selected autoplay loop controls');
   const nonEmptyElementsMap = createLookupTable('non_empty_elements', 'td th iframe video audio object ' +
     'script pre code', shortEndedElementsMap);
-  const moveCaretBeforeOnEnterElementsMap = createLookupTable('move_caret_before_on_enter_elements', 'table', nonEmptyElementsMap);
+  const moveCaretBeforeOnEnterElementsMap = createLookupTable('move_caret_before_on_enter_elements', 'td th igrame video audio object ' +
```

Comment:
`igrame` isn't an element as far as I know? :stuck_out_tongue:

Reply:
Also to remove some duplication, should we create something like a `base` list and use it for both this and `nonEmptyElementsMap`?